docs/src/snap/howto/cos-lite.md

```diff
@@ -0,0 +1,142 @@
+# Integrating with COS Lite
+
+**Canonical Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
```

Comment:
```suggestion
It is advisable to have a monitoring solution which will run whether the
cluster itself is running or not. It may also be useful to integrate monitoring
into existing setups.
```